sale_tier_validation/views/sale_order_view.xml

```diff
@@ -23,4 +23,13 @@
             </filter>
         </field>
     </record>
+    <record id="view_order_form_hide_send_email" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_quotation_send']" position="attributes">
+                <attribute name="attrs">{'invisible': ['|','|',('need_validation','=',True),'&amp;',('validated','=',False),('need_validation','=',False),'&amp;',('validated','=',False)]}</attribute>
```

Comment:
Is `('validated','=',False),('need_validation','=',False)` necessary?

Reply:
hmm, i think so. i was not able to use the validated = false because i didnt add it to view as invisible. let me try and check again. thanks Kanda san

Reply:
@rinaldifirdaus What about the case where validation is not required in the first place (according to the tier setting)? Not sure such case exists at the client, but we should consider this case if we are to update the OCA module.